website_event_questions_template/i18n/es.po

```diff
@@ -0,0 +1,185 @@
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_name
+msgid "Name"
+msgstr "Nombre"
+
+#. module: website_event_questions_template
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_answer_question_id
+#, fuzzy
+msgid "Question id"
+msgstr "Id cuenta"
+
+#. module: website_event_questions_template
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_question_ids
+#: model:ir.ui.view,arch_db:website_event_questions_template.view_event_question_template_form
+msgid "Questions"
+msgstr "Preguntas"
+
+#. module: website_event_questions_template
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_config_settings_event_question_template_id
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_event_event_question_template_id
+msgid "Questions Template"
+msgstr "Plantilla de preguntas"
+
+#. module: website_event_questions_template
+#: model:ir.ui.menu,name:website_event_questions_template.menu_event_question_template
+msgid "Questions Templates"
+msgstr "Plantilla de preguntas de eventos"
+
+#. module: website_event_questions_template
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_answer_sequence
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_question_sequence
+msgid "Sequence"
+msgstr "Secuencia"
+
+#. module: website_event_questions_template
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_question_title
+msgid "Title"
+msgstr "Título"
+
+#. module: website_event_questions_template
+#: code:addons/website_event_questions_template/models/event.py:29
+#, python-format
+msgid "You can not load a template if there are defined questions"
+msgstr "No puedes cargar una plantilla el evento ya tiene preguntas definidas"
+
+#. module: website_event_questions_template
+#: model:ir.model,name:website_event_questions_template.model_event_config_settings
+msgid "event.config.settings"
+msgstr "Abrir menú 'Configuración'"
+
+#. module: website_event_questions_template
+#: model:ir.model,name:website_event_questions_template.model_event_question_template
+#, fuzzy
+msgid "event.question.template"
+msgstr "Plantilla de preguntas de eventos"
+
+#. module: website_event_questions_template
+#: model:ir.model,name:website_event_questions_template.model_event_question_template_answer
+#, fuzzy
+msgid "event.question.template.answer"
+msgstr "Responder pregunta relativa al puesto"
+
+#. module: website_event_questions_template
+#: model:ir.model,name:website_event_questions_template.model_event_question_template_question
+#, fuzzy
+msgid "event.question.template.question"
+msgstr "Plantilla de preguntas de eventos"
+
+#~ msgid "Question Template"
```

Comment:
Why is this commented? Other side effect of POEdit?

Reply:
What app are you using to translate?

Reply:
Text editor :wink:

Reply:
If you're a human, you can use POEdit's *Catalog > Purge unused translations* instead 😋